guardian/support-frontend
# feat: add payment method validation

## What are you doing in this PR?

Add some validation for when a user hasn't selected a payment method. However due to the way we're using HTML validation, this will only kick in after the other form fields have been validated (by native validation).



[**Trello Card**](https://trello.com/c/TtB5PxAJ/1135-generic-one-time-checkout-payment-validation)

## Why are you doing this?

This validation existed on the old checkout - and provides a better user experience if someone tries to press the button without having selected a payment radio.



## How to test

To go generic or one time checkout, fill out personal details. Do not select a payment method. Click payment CTA. See "please select a payment method" validation.



## Screenshots
![image](https://github.com/user-attachments/assets/17609ce7-6c24-40c1-9964-749de83eceeb)
